src/components/OrderBookChart/index.tsx

```diff
@@ -147,11 +159,10 @@ export const Chart: React.FC<ChartProps> = props => {
 
         const initialZoom = calcInitialZoom(bids, asks)
 
+        // Zoom in, only if this is the first load of this token pair
+        adjustZoomOnFirstLoad(initialZoom)
```

Comment:
ah by token pair, could also be done with an effect based on token choice i guess. anyways this works too